doc/releases/changelog-dev.md

```diff
@@ -18,6 +18,9 @@
 
 <h3>Breaking changes 💔</h3>
 
+* ``QuantumScript.to_openqasm`` has been removed. Please use ``qml.to_openqasm`` instead.
```

Comment:
Thank you @comp-phys-marc!

Would it be possible to slightly update the changelog to include something about why the change was made, or the benefit for the user? One suggestion would be something along the lines of:
The ``QuantumScript.to_openqasm`` method has been removed to remove redundancies in the code base. Please use ``qml.to_openqasm`` instead to convert a circuit to OpenQASM.

Other than this LGTM!